src/abstract.jl

```diff
@@ -47,42 +50,59 @@ mutable struct LinearOperator{T, I <: Integer, F, Ft, Fct} <: AbstractLinearOper
   nprod::I
   ntprod::I
   nctprod::I
+  args5::Bool
+  use_prod5!::Bool # true for 5-args mul! and for composite operators created with operators that use the 3-args mul!
+  Mv5::S
+  Mtu5::S
+  allocated5::Bool # true for 5-args mul!, false for 3-args mul! until the vectors are allocated
+end
+
+function LinearOperator{T}(nrow::I, ncol::I, symmetric::Bool, hermitian::Bool, 
+                  prod!::F, tprod!::Ft, ctprod!::Fct,
+                  nprod::I, ntprod::I, nctprod::I,
+                  ) where {T,I<:Integer,F,Ft,Fct}
+                  
+  Mv5, Mtu5 = T[], T[]
```

Comment:
It will only works on CPU for information.

Reply:
It will still work on GPU I guess until you call the 5-args `mul!` with a 3-args operator

Reply:
I only use the 3-args mul! on GPU, so it should work fine :smiley: 
I open a PR for a release `2.2.3`.

Reply:
Something to mention in the docs